app/models/mod_note.rb

```diff
@@ -209,7 +209,7 @@ def self.tattle_on_traffic_attribution!(story)
   # and save attempts, leading to duplicate notes
   def self.create_without_dupe!(attrs)
     latest = attrs[:user].mod_notes.last
-    return latest if latest && attrs[:moderator] == latest.moderator && attrs[:note] == latest.note
+    return latest if latest && attrs[:moderator] == latest.moderator && attrs[:note].strip == latest.note
```

Comment:
However if `create_without_dupe!` works fine it should prevent creating duplicate mod notes, then i started digger further, i found there is:

https://github.com/lobsters/lobsters/blob/407a212f68fb1e047766725a9a39fe9cf97bdfb8/app/models/mod_note.rb#L33-L36

which makes `attrs[:note] == latest.note` will never be true. :disappointed:

Reply:
Thank you for running down this bug, it has irritated me for years.

I think I'd rather you remove the `.strip` from `note=` than try to replicate it everywhere. I can't guess why I added it in the first place; the whitespace will either be stripped by the markdown processor or ignored by the browser.